### PR TITLE
raise AuditLogRegistrationError on invalid app in settings

### LINKED ISSUE #491: backtrace from typo in app name
I got the following backtrace when I made a typo in an app name in `AUDITLOG_INCLUDE_TRACKING_MODELS`

```
 Traceback (most recent call last):
  File "/home/runner/work/rubberboot-admin-web/rubberboot-admin-web/code/src/manage.py", line 22, in <module>
    main()
  File "/home/runner/work/rubberboot-admin-web/rubberboot-admin-web/code/src/manage.py", line 18, in main
    execute_from_command_line(sys.argv)
  File "/opt/hostedtoolcache/Python/3.10.9/x64/lib/python3.10/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/opt/hostedtoolcache/Python/3.10.9/x64/lib/python3.10/site-packages/django/core/management/__init__.py", line 395, in execute
    django.setup()
  File "/opt/hostedtoolcache/Python/3.10.9/x64/lib/python3.10/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/opt/hostedtoolcache/Python/3.10.9/x64/lib/python3.10/site-packages/django/apps/registry.py", line 122, in populate
    app_config.ready()
  File "/opt/hostedtoolcache/Python/3.10.9/x64/lib/python3.10/site-packages/auditlog/apps.py", line 12, in ready
    auditlog.register_from_settings()
  File "/opt/hostedtoolcache/Python/3.10.9/x64/lib/python3.10/site-packages/auditlog/registry.py", line 322, in register_from_settings
    self._register_models(settings.AUDITLOG_INCLUDE_TRACKING_MODELS)
  File "/opt/hostedtoolcache/Python/3.10.9/x64/lib/python3.10/site-packages/auditlog/registry.py", line 263, in _register_models
    model["model"] = self._get_model_classes(model["model"])[0]
```

The issue is in this line https://github.com/jazzband/django-auditlog/blob/ffa6d34b111d3ff84667a85b91fa48c4bfbb3471/auditlog/registry.py#L264 where it is assumed that a non-empty list is returned from `self._get_model_classes()` but that method can return an empty list in case of a `LookupError` exception.

I propose in case an unknown app is used in settings we either failing with a nicer error message, or skip the entry.

### PR DESCRIPTION
**Pull request by tykling**
Includes a test to trigger the issue. Fixes #491

### CODE REVIEW

File: auditlog/registry.py [hunk 1]
```diff
@@ -261,7 +261,12 @@ def _register_models(self, models: Iterable[Union[str, Dict[str, Any]]]) -> None
                     self.unregister(model_class)
                     self.register(model_class)
             elif isinstance(model, dict):
-                model["model"] = self._get_model_classes(model["model"])[0]
+                appmodel = self._get_model_classes(model["model"])
+                if not appmodel:
+                    raise AuditLogRegistrationError(
+                        f"An error was encountered while registering model {model['model']} - make sure the app is registered correctly."
+                    )
```
Comment (hramezani) [inline auditlog/registry.py:268]:
```suggestion
                    raise AuditLogRegistrationError(
                        f"An error was encountered while registering model '{model['model']}'."
                        "make sure the app is registered correctly."
                    )
```

File: auditlog_tests/tests.py [hunk 2]
```diff
@@ -1158,6 +1158,18 @@ def test_register_from_settings_invalid_settings(self):
             ):
                 self.test_auditlog.register_from_settings()
 
+        with override_settings(
+            AUDITLOG_INCLUDE_TRACKING_MODELS=({"model": "notanapp.test"},)
+        ):
+            with self.assertRaisesMessage(
+                AuditLogRegistrationError,
+                (
+                    "An error was encountered while registering model notanapp.test"
```
Comment (hramezani) [inline auditlog_tests/tests.py:1167]:
```suggestion
                    "An error was encountered while registering model 'notanapp.test'"
```

**Review by hramezani — APPROVED** [review]
LGTM :+1:

**Comment by hramezani** [comment]
Thanks @tykling for the fix.

@alieh-rymasheuski Do you want to take a look?

**Review by alieh-rymasheuski — APPROVED** [review]

**Pull request merged**